### ISSUE TITLE
Penguin Watch: pan & zoom doesn't work from the keyboard

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#3858]
## Package
lib-classifier

## Describe the bug
The keyboard controls for zooming and panning should be available while you are placing marks with a trackpad or mouse.

## To Reproduce
1. Go to https://frontend.preview.zooniverse.org/projects/penguintom79/penguin-watch/classify/workflow/21738
2. Place a mark.
3. Press the + key to zoom in.

## Expected behavior
- go to https://www.penguinwatch.org
- navigate to the Classify page.
- Place a mark on the photo, then zoom in with the + key and pan with the cursor keys.
